docs/btcli.md

```diff
@@ -1056,16 +1049,13 @@ $ btcli stake child revoke [OPTIONS]
 
 #### btcli stake child set
 
-Add children hotkeys on a specified subnet on the Bittensor network.
+Set child hotkeys on a specified subnet on the Bittensor network. This command will **overwrite** previous child hotkey settings. Use this command only if you want to change the child hotkeys. 
```

Comment:
> Use this command only if you want to change the child hotkeys. 
I might change this wording to ensure people who haven't set child keys yet that this is the way to set them, but it is also used for changing your child key list if you already have children.  doesn't have to be that wordy, but something that gets that point across.